### PR TITLE
Standalone

### PR DESCRIPTION
**Pull request by jbygdell**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [x] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

Modifies the entrypoint script to generate a federated or stand alone configuration based on if the `CEGA_CONNECTION` ENV is defined or not.
### Changes made:

1) Extended the entrypoint script to generate a stand alone config
2) Fixed the chmod command for the certificates
3) Renamed the entrypoint script from ega-entrypoint to docker-entrypoint

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -145,5 +221,164 @@ cat > "/var/lib/rabbitmq/advanced.config" <<EOF
...
+      "arguments": {},
+      "destination": "files.inbox",
+      "routing_key": "files.inbox"
+    },
+    {
+      "source": "lega",
+      "vhost": "${MQ_VHOST:-/}",
+      "destination_type": "queue",
+      "arguments": {},
+      "destination": "files.processing",
```
Comment (blankdots) [inline entrypoint.sh:366]: not sure how we would make use of this queue ... :thinking:

**Review by blankdots — APPROVED** [review]
for standalone we might need just `files.inbox`, (maybe) `files`, `errors`, `stableIDs` and `completed`.

We need to modify hard-coded publish of `errors` to `cega` exchange in neicnordic/LocalEGA, or we can create a binding for it to route internally, either way it think we need to make changes to neicnordic/LocalEGA code.

a biding such as:
```
{
      "source": "cega",
      "vhost": "${MQ_VHOST:-/}",
      "destination": "errors",
      "destination_type": "queue",
      "routing_key": "errors",
      "arguments": {}
    }
```
This being said, we could approve as is and change later as we change the code for neicnordic/LocalEGA

File: entrypoint.sh (hunk 1 above)
Comment (blankdots) [reply entrypoint.sh:366]: we can come back to it later.

**Comment by jbygdell** [comment]
There are a lot of hard coded stuff in the python code that needs to be looked at.

**Comment by blankdots** [comment]
tracked here: https://github.com/neicnordic/LocalEGA/issues/33

**Pull request merged**